converters/fromZigbee.js

```diff
@@ -2752,6 +2773,61 @@ const converters = {
             return results;
         },
     },
+    heiman_ir_remote: {
+        cluster: 'heimanSpecificInfraRedRemote',
+        type: ['commandStudyKeyRsp', 'commandCreateIdRsp', 'commandGetIdAndKeyCodeListRsp'],
+        convert: (model, msg, publish, options, meta) => {
+            switch (msg.type) {
+            case 'commandStudyKeyRsp':
+                return {
+                    learn: {
```

Comment:
@Koenkk 
Actually, after publishing `learn` and `create` results I need to publish empty payload for them, like `action`. Could you point me, how to publish them from converter?

Reply:
Instead of returning an object, return e.g. this:

```js
{
 action: "learn",
 action_key_code:  msg.data.keyCode,
 action_result: msg.data.result === 1 ? 'success' : 'error',
 action_id: msg.data.result === 1 ? msg.data.id : undefined
}
```

Reply:
May be better to something like this, to prevent old data from previous action?
```
{
 action: "learn",
 action_data: {
   key_code:  msg.data.keyCode,
   result: msg.data.result === 1 ? 'success' : 'error',
   id: msg.data.result === 1 ? msg.data.id : undefined
 }
}
```

Reply:
Ok, I found other examples in homeassistant.js was prefixed with `action_`